src/events/ready.ts

```diff
@@ -1,10 +1,15 @@
 import { REST } from "@discordjs/rest";
 import { Routes } from "discord-api-types/v9";
 import {
+<<<<<<< Updated upstream
   Client,
   MessageActionRow,
   MessageButton,
   MessageEmbed
+=======
+  ActionRowBuilder, ApplicationCommandPermissionType, ButtonBuilder, ButtonStyle, ChannelType,
+  Client, EmbedBuilder
+>>>>>>> Stashed changes
```

Comment:
See above

Reply:
@Juilzze resolved this issue in latest commit.